### ISSUE TITLE
Multiple tooltip styles?

### ISSUE THREAD

**Issue by acomito** [lookback/meteor-tooltips#14]
Is it possible to have multiple styles for tooltips without a way to set a class on the tooltip?

For instance, if I need red tooltips in one part of the app and blue tooltips in another?

**Comment by johanbrook** [comment]
Good idea! IMO it's easiest to provide a way of setting custom classes (styles) on a specific tooltip:

``` html
<span data-tooltip="Hello" data-tooltip-class="warning large">..</span>
```

What do you think?

**Comment by acomito** [comment]
That would be awesome!

Ive seen other packages use that method 

(e.g. https://github.com/angular-ui/bootstrap/tree/master/src/tooltip)

**Comment by acomito** [comment]
Hi!

Wanted to check in to see if there was any plans to implement this feature in the near future?

**Comment by johanbrook** [comment]
It could be: PRs are welcomed :)

**Comment by acomito** [comment]
I will try
